### ISSUE TITLE
Imort & Export option for Favourites

### ISSUE THREAD

**Issue by shuvashish76** [VishnuSanal/Quotes#60]
Option to import & export favourite quotes in various file formats.

**Comment by VishnuSanal** [comment]
bumping up the priority https://t.me/QuotesStatusCreator/5542

**Comment by Quantum-Future** [comment]
Agree, it could be a useful feature for making backups and transfer info to other phone.
Also for importing quotes I have in a .TXT
